OpenRA.Mods.Common/Graphics/DefaultSpriteSequence.cs

```diff
@@ -41,49 +41,49 @@ public bool TryGetPaletteForFrame(int frame, out uint[] palette)
 
 	public class DefaultSpriteSequenceLoader : ISpriteSequenceLoader
 	{
+		static readonly MiniYaml NoData = new MiniYaml(null);
```

Comment:
This breaks `ExtractSpriteSequenceDocsCommand` ☹️ 
Please change
```diff
Properties = type.GetFields(BindingFlags.NonPublic | BindingFlags.Static)
-	.Where(fi => fi.FieldType.GetGenericTypeDefinition() == typeof(SpriteSequenceField<>))
+	.Where(fi => fi.FieldType.IsGenericType && fi.FieldType.GetGenericTypeDefinition() == typeof(SpriteSequenceField<>))
	.Select(fi =>
```
in `ExtractSpriteSequenceDocsCommand`, line 65.